packages/site/cms-site/src/image/Image.tsx

```diff
@@ -3,17 +3,22 @@ import NextImage, { ImageLoaderProps, ImageProps as NextImageProps } from "next/
 
 // Fallback to 1 / 1 aspect ratio for invalid value format
 export function parseAspectRatio(value: string | number): number {
-    value = String(value);
     let width, height;
-    if (!Number(value)) {
+    if (typeof value === "string") {
         [width, height] = value.split(/[x/:]/).map((v) => {
             let ret: number | undefined = parseFloat(v);
-            if (isNaN(ret)) ret = undefined;
+            if (isNaN(ret) || isNaN(parseInt(v))) ret = undefined;
```

Comment:
Why do you need to parseInt as well as parseFloat?

Reply:
ParseInt does not allow the value Infinity while parseFloat() and Number() do